pkgs/os-specific/linux/powertop/default.nix

```diff
@@ -8,10 +8,11 @@ stdenv.mkDerivation rec {
     sha256 = "0nlwazxbnn0k6q5f5b09wdhw0f194lpzkp3l7vxansqhfczmcyx8";
   };
 
-  buildInputs = [ gettext libnl ncurses pciutils pkgconfig zlib ];
+  buildInputs = [ gettext libnl ncurses pciutils pkgconfig zlib xorg.xset ];
 
   postPatch = ''
     substituteInPlace src/main.cpp --replace "/sbin/modprobe" "modprobe"
+    substituteInPlace src/calibrate/calibrate.cpp --replace "/usr/bin/xset" "xset"
```

Comment:
`xet` is, in general, not available in `$PATH`, so this patch works only for those people who have the utility installed in their environment. It would be great if you could implement the changes suggested in https://github.com/NixOS/nixpkgs/pull/12707/files#r51353614.